.github/workflows/python-app.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest
+        pip install flake8 pytest pytest-mock
```

Comment:
Does pytest-mock need to go in requirements.txt as well?

Reply:
No, because it's not a requirement for `adatest`, only the tests. I was actually going to propose that we add a `requirements-dev.txt` file too, for just this purpose.